core/templates/dev/head/services/AssetsBackendApiService.js

```diff
@@ -143,8 +177,14 @@ oppia.factory('AssetsBackendApiService', [
       });
     };
 
-    var _isCurrentlyBeingRequested = function(filename) {
-      return _filesCurrentlyBeingRequested.some(function(request) {
+    var _isAudioCurrentlyBeingRequested = function(filename) {
+      return _audioFilesCurrentlyBeingRequested.some(function(request) {
+        return request.filename === filename;
+      });
+    };
+
+    var _isImageCurrentlyBeingRequested = function(filename) {
```

Comment:
In all previous private (_*) methods you use the style `_method = function(something, something, assetType)`. Here you have two separate methods for audio and image, is there any specific reason for this?

If no, I think we should be consistent and the method should be something like `_isAssetCurrentlyBeingRequested(filename, assetType)`

Reply:
Done!